pkg/sdn/plugin/controller.go

```diff
@@ -57,31 +52,32 @@ func alreadySetUp(multitenant bool, localSubnetGatewayCIDR, clusterNetworkCIDR s
 	var found bool
 
 	exec := kexec.New()
-	itx := ipcmd.NewTransaction(exec, LBR)
-	addrs, err := itx.GetAddresses()
+	itx := ipcmd.NewTransaction(exec, TUN)
+	routes, err := itx.GetRoutes()
 	itx.EndTransaction()
 	if err != nil {
 		return false
 	}
 	found = false
-	for _, addr := range addrs {
-		if addr == localSubnetGatewayCIDR {
+	for _, route := range routes {
+		if strings.Contains(route, clusterNetworkCIDR+" ") {
```

Comment:
is there any reason you swapped the order of the localSubnetGatewayCIDR and clusterNetworkCIDR checks?